models/issue.go

```diff
@@ -1354,6 +1359,15 @@ func applyReviewRequestedCondition(sess *xorm.Session, reviewRequestedID int64)
 			reviewRequestedID, ReviewTypeApprove, ReviewTypeReject, ReviewTypeRequest, reviewRequestedID)
 }
 
+func applySubscribedCondition(sess *xorm.Session, subscriberID int64) *xorm.Session {
+	return sess.And("NOT issue.id IN (SELECT issue_id FROM issue_watch WHERE is_watching = ? AND user_id = ?)", false, subscriberID).
```

Comment:
It would be better to use:
```suggestion
	return sess.And("issue.id NOT IN (SELECT issue_id FROM issue_watch WHERE is_watching = ? AND user_id = ?)", false, subscriberID).
```